specs/UserActivityHistory/UserActivityHistoryAPI.md

```diff
@@ -0,0 +1,243 @@
+UserActivityHistory
+===
+
+# Background
+
+The UserActivity class can be used to note down and preserve a record of activities that the user
+is currently doing on their computer - e.g., browsing a website, reading a Word document, etc.
+
+To record user activity, you use UserActivityChannel to retrieve a UserActivity object via the
+API GetOrCreateUserActivityAsync. If a UserActivity with the given ID already exists, it will be
+returned; otherwise, a new UserActivity object will be created and returned. You can then call
+the API GetSession to return a UserActivitySession object that tracks how long the user is engaged
+in that activity. This structure allows multiple sessions to be associated with the same activity,
+representing the case where the user completes that activity a bit at a time - e.g., beginning to
+watch a movie, then pausing, then watching more later. These will be treated as the same singular
+user activity that spans multiple sessions.
+
+UserActivityHistory is a new set of APIs that allow you to query the past 28 days of the user's
```

Comment:
28 days --> is this a parameter of how the history is stored? e.g. can I say how long to keep this for? (maybe with a system-defined upper limit)?

Reply:
28 days is an arbitrarily-defined length of time that seemed like an amount that would allow users to accomplish everything they want to do without storing more than would be required.  There isn't anything in the current design that would allow users to set it to a different amount, though if we wanted to add something to settings, there also isn't anything that would prevent us from doing that.

I can add an "up to" comment to cover the possibility of us adding a setting later.